third_party/move/move-compiler-v2/src/pipeline/ability_checker.rs

```diff
@@ -6,7 +6,8 @@
 //! - liveness analysis and lifetime analysis have been performed
 //! - Copies and moves have been made explicit in assignment instructions
 
-use move_binary_format::file_format::{Ability, AbilitySet};
+use super::abort_analysis::{AbortStateAnnotation, AbortStateAtCodeOffset};
```

Comment:
It looks like the general convention here is to use `crate::full_path`.

Reply:
Done. (was imported by vscode)